src/Model/FieldType/BooleanField.test.ts

```diff
@@ -43,4 +45,9 @@ describe("FieldType: BooleanField", () => {
   it("should be serialized", () => {
     expect(BooleanField.serialize).to.be.true;
   });
+
+  it("normalizes any value into a boolean", () => {
+    const value = "true";
```

Comment:
Do we want a couple more test cases to show that the `Boolean` function is not parsing the string for the value "true" but is using the standard rules for 0, "", NaN, etc.?

Reply:
Good idea!